controllers/articles.controllers.js

```diff
@@ -19,13 +19,20 @@ exports.getArticle = (request, response, next) => {
       response.status(200).send({ article });
     })
     .catch((err) => {
-      next(err)
+      next(err);
     });
 };
 
 exports.getArticles = (request, response, next) => {
   const { article_id } = request.params;
-  const promises = [readArticles(article_id)];
+  const { topic } = request.query;
+  console.log(request.query, 'request query')
+  console.log(topic, 'topic in cont')
+  const { sort_by } = request.query;
+  console.log(sort_by, 'sort by in cont')
+  const { order } = request.query;
+  console.log(order, 'order in cont')
```

Comment:
I would recommend doing a search for console.log in vsc then just deleting all instances of it